src/CLI/actioner/sonic-cli-neighbors.py

```diff
@@ -211,8 +211,12 @@ def run(func, args):
         elif 'sonic-neighbor:output' in response.keys():
             status = response['sonic-neighbor:output']
             status = status['response']
-            if (status != "Success"):
+            if "255" in status:
```

Comment:
What's 255 in status, did we mention anything about 255 in YANG, please fix it appropriately.